packages/docusaurus-1.x/lib/static/css/main.css

```diff
@@ -1975,6 +1975,10 @@ input::placeholder {
   color: #717171;
 }
 
+.onPageNav .toc-headings>li>a.active {
```

Comment:
Please follow the coding style in the rest of this CSS file in terms of spacing and indentation.